# ChIPseeker-Analysis-of-ChIP-seq-data-in-R

ChIPseeker is a very useful R package for carrying out detailed analyses on Chromatin Immunoprecipitation (ChIP)- sequencing data that lists the genomic intervals or "peaks" for a transcription factor binding to the genome. This hands-on session will have two parts:

### First 30-minute-session:

Get used to reading and understanding documentation (manual) for R packages<br/> 
Learn how to create objects and variables in R<br/>
Learn scripting syntax (conventions) used in the ChIPseeker package<br/>

### Second 2-hour session:

Summarize & visualize important information about the peaks<br/> 
Annotate the peaks with gene ID as well as genomic location<br/>
Carry out functional enrichment analysis on the peaks<br/>

For further information, please refer to:<br/>
G Yu, LG Wang, QY He. ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparison and visualization. Bioinformatics 2015, 31(14):2382-2383<br/>
[Link to paper](http://dx.doi.org/10.1093/bioinformatics/btv145)
[Link to vignette](https://bioconductor.org/packages/release/bioc/vignettes/ChIPseeker/inst/doc/ChIPseeker.html#session-information)



