# ChIPseeker-Analysis-of-ChIP-seq-data-in-R

ChIPseeker is a very useful R package for carrying out detailed analyses on Chromatin Immunoprecipitation (ChIP)- sequencing data that lists the genomic intervals or "peaks" for a transcription factor binding to the genome. This hands-on session will have two parts:

### First 30-minute-session: Foundations 

Understand the context of Chipseeker by taking a brief look at BioConductor, identifying documentation sources, and covering foundational R concepts and syntax.<br/> 
* BioConductor: 
  * Bioconductor provides tools for the analysis and comprehension of high-throughput genomic data. 
  * Bioconductor uses the R statistical programming language, and is open source and open development. 
  * It has two releases each year, and an active user community.
  * All Bioconductor packages can be found here: https://bioconductor.org/packages/release/BiocViews.html#___Software
* ChIPseeker documentation:https://bioconductor.org/packages/release/bioc/manuals/ChIPseeker/man/ChIPseeker.pdf 
* Foundational R for Chipseeker: [click here to view the R markdown file](https://sauuyer.github.io/intro-to-r-for-seurat/r-intro-chipseeker.html) 


### Second 2-hour session: Using ChIPseeker

Summarize & visualize important information about the peaks<br/> 
Annotate the peaks with gene ID as well as genomic location<br/>
Carry out functional enrichment analysis on the peaks<br/>

For further information, please refer to:<br/>
G Yu, LG Wang, QY He. ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparison and visualization. Bioinformatics 2015, 31(14):2382-2383<br/>
[Link to paper](http://dx.doi.org/10.1093/bioinformatics/btv145)<br/>
[Link to vignette](https://bioconductor.org/packages/release/bioc/vignettes/ChIPseeker/inst/doc/ChIPseeker.html#session-information)

* [R script for analysis](https://github.com/Nur-Taz/ChIPseeker-Analysis-of-ChIP-seq-data-in-R/blob/master/Analysis_example_ChIPseeker.R)


